go.mod

```diff
@@ -34,4 +35,6 @@ require (
 	sigs.k8s.io/controller-runtime v0.8.3
 )
 
+replace github.com/codeready-toolchain/api => github.com/xcoulon/api v0.0.0-20211110143945-dec71f4e5073
```

Comment:
reminder

Reply:
done in https://github.com/codeready-toolchain/toolchain-common/pull/216/commits/d991b0a90898933c3f04346184e8111d8e24693a